lib/mp4/transmuxer.js

```diff
@@ -862,6 +864,14 @@ CoalesceStream.prototype.flush = function(flushSource) {
     timelineStartPts = this.videoTrack.timelineStartInfo.pts;
   } else if (this.audioTrack) {
     timelineStartPts = this.audioTrack.timelineStartInfo.pts;
+    ['audioobjecttype',
+     'channelcount',
+     'samplerate',
+     'samplingfrequencyindex',
+     'samplesize'].forEach(function(audioProp) {
+       event.info[audioProp] = self.audioTrack[audioProp];
+     });
```

Comment:
`forEach` takes a second parameter as the context the function is given when executed.
```
[...].forEach(function(audioProp) {
       event.info[audioProp] = this.audioTrack[audioProp];
     }, this);
```